src/server/api/routers/requirements.ts

```diff
@@ -0,0 +1,84 @@
+import { z } from "zod";
+import {
+  createTRPCRouter,
+  protectedProcedure,
+  publicProcedure,
+} from "techme/server/api/trpc";
+import { requirements } from "techme/server/db/schema";
+import { eq } from "drizzle-orm";
+
+export const requirementsRouter = createTRPCRouter({
+  // Create a new requirement
```

Comment:
```suggestion```
remove unnecessary comments. code is self explanatory

Reply:
as a general coding sense comments should be added to explain *why* you are making the changes not what